### ISSUE TITLE
Ch 4, adding in teaching students to change shape in addition to color

### ISSUE THREAD

**Issue by Zibiana** [VectorPosse/intro_stats#80]
Would it be possible to add an example that uses shape and color in the scatterplots? The color can be hard to tell the difference for some types of color blindness and makes it hard to answer some of the later questions.

**Comment by VectorPosse** [comment]
Yes, good idea. I am also going to add the viridis theme throughout for a
more colorblind friendly palette in all graphs. The shape thing will also
be helpful for those looking to publish in journals that only accept B&W.

**Comment by VectorPosse** [comment]
Wow, it turns out that the shape aesthetic is a bit of a minefield. The defaults are truly awful. However, this is an important skill, so I've added some content to Ch. 4. It may not be worth the effort to continue to use shapes throughout the book. Hopefully, the consistent use of the Viridis palette will make the shape issue less of a big deal as the graphs will now be colorblind-friendly without needing to be explicitly black-and-white.
